app/src/main/java/com/afwsamples/testdpc/common/RestrictionManagerCompat.java

```diff
@@ -31,6 +31,12 @@
 public class RestrictionManagerCompat {
```

Comment:
@AndrewScull  + Andrew to the discussion ☺️ 

Hmm, speaking of solution 1, I am not quite understand why putting RestrictionEnrty into the fragment argument bundle requires conversion to bundle. RestrictionEnrty is parcelable, and should be ok to put in the arguments bundle straight away?

```
        arguments.putBundle(ARG_INITIAL_VALUE, bundle);
        arguments.putparcelableArray(ARG_RESTRICTIONS, restrictions);

```

Thanks!